docs/cities/manchester/roles-and-duties.md

```diff
@@ -0,0 +1,111 @@
+---
+id: roles-and-duties
+title: Roles & Duties
+---
+
+Under the main trinity of [Volunteer teams](volunteers/teams.md),namely Outreach, Education & Personal Development,
+generally, we need to act and fulfill different roles in order to run the course smoothly and effectively.
+These roles also help to keep each member being engaged and organised.
+
+A volunteer can wear different hats on the way depending on the schedule and agenda, or maybe special circumstances.
+All roles and tasks are optional, and no one is forced, or coerced into to doing things that they do not feel comfortable with.
+
+Any help is welcomed :) 
+
+## Roles
+Mostly, we need the following roles as below.
+Each role is charged of duties and demands various skill set.
```

Comment:
Suggest...

"The duties for each role vary, and will require a diverse skill set."